pyomo/contrib/mindtpy/initialization.py

```diff
@@ -124,18 +131,22 @@ def init_rNLP(solve_data, config):
         dual_values = list(
             m.dual[c] for c in MindtPy.constraint_list) if config.use_dual else None
         # Add OA cut
-        if main_objective.sense == minimize:
+        if main_objective.sense == minimize and not math.isnan(results['Problem'][0]['Lower bound']):
```

Comment:
I would add a comment here mentioning that this covers the case when the Lower bound does not exist.